### ISSUE TITLE
Make the acknowledgement (checkbox) of data use conditions one step action

### ISSUE THREAD

**Issue by jennytdtran** [elixir-luxembourg/data-catalog#23]
Currently when requesting access to a dataset in Data Catalog, one has to tick each use restrictions one by one. For some datasets, the use restrictions consist of up to 20 conditions. Consider having just one tickbox for all.

**Comment by vildead** [comment]
Indeed. This is especially annoying in cases when users are requesting multiple datasets in the row.
